app/services/account_reset/create_request.rb

```diff
@@ -46,11 +48,23 @@ def notify_user_by_sms_if_applicable
       @telephony_response = Telephony.send_account_reset_notice(
         to: phone,
         country_code: Phonelib.parse(phone).country,
+        interval: confirmation_period,
       )
     end
 
     def extra_analytics_attributes
       @telephony_response&.extra&.slice(:request_id, :message_id) || {}
     end
+
+    def confirmation_period
+      current_time = Time.zone.now
+
+      distance_of_time_in_words(
+        current_time,
+        current_time + Devise.confirm_within,
```

Comment:
Should this be using `account_reset_wait_period_days` config instead of `Devise.confirm_within` ?

Reply:
Yes, this should use `account_reset_wait_period_days` instead of `Devise.confirm_within`. I have also renamed `confirmation_period` to `account_reset_wait_period`